part1/readme.md

```diff
@@ -0,0 +1,99 @@
+## Installing Stacker
+You can install stacker via pip
+
+```
+pip install stacker stacker_blueprints
+```
+and you can verify that the installation worked by running the stacker command in your shell
+
+```
+$ stacker
+
+usage: stacker [-h] [--version] {build,destroy,info,diff} ...
+stacker: error: too few arguments
+```
+
+## Credentials
+
+In order for Stacker to work we must give it AWS credentials in some way, there are several places where stacker checks for credentials. 
+
+1. The `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables
+2. Shared credential file (*~/.aws/credentials*)
+3. AWS config file (*~/.aws/config*)
+4. You can specify a role using `AWS_PROFILE`
+
+The easiest way to get your environment configured is to just use:
+`aws configure` to get it working. 
+
+## Setting Up
+
+When running a build with stacker we will need to create 2 files, an environment file and an configuration file.
+
+``` 
+touch infra.yaml prod.env
+```
```

Comment:
A naïve user could think that the files need to be laid out in some special dir, so I'd explain that they can be anywhere, and suggest using an empty dir.